content/kit/batteries/charging.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Battery Charging
+---
+
+The charging cable is cable-tied to the charger and you must not remove this cable. The charging lead is terminated with 4mm bullet connectors, which present a very high risk of shorting the battery if removed from the charger.
+
+## Charging Checklist
+This list must be followed carefully each time a battery is connected to the charger.
+
+1. Plug in the mains adapter and plug it into the battery charger.
+2. Connect the charging lead from the charger to the battery. Ensure you plug the lead into the charger first and the battery second to avoid risk of shorting the battery.
+3. Connect the monitoring lead from the battery to the charger.
+4. Place the battery in the charging bag.
+5. Configure the charger using the menu system as follows. 
+
+### Starting Charging
+
+{{% notice warning %}}
+Do not attempt to charge your battery if any of the cables are lose or frayed. Contact us immediately if you think there's an issue!
+{{% /notice %}}
+
+1. Press `Start/Enter` to select lithium battery charging.
+2. Press `Start/Enter` to edit the charging current.
+3. The current (bottom left) should start blinking. Use the `Inc./Dec.` buttons until it reads 2.2A. Press `Start/Enter` to store the value and to begin editing the battery voltage. Use the `Inc./Dec.` buttons until it reads 11.1V(3S).
+4. Hold `Start/Enter` for 3 seconds to confirm. The charger will try to detect the battery.
+5. Once the battery has been detected, the screen will show _CONFIRM (ENTER)_. Press `Start/Enter` to start charging.
+
+{{% notice warning %}}
+The important thing to note here is that the number of cells detected by the charger (_R_) and the number of cells set by you (_S_) are both 3. *If either of these numbers are different immediately power down the charger and contact us.*
+{{% /notice %}}
+
+The screen shows the charging current, the current voltage, the time elapsed and the energy provided to the battery.
+
+### Stopping Charging
+The charger will beep when the battery is fully charged. You must then:
+
+1. Press `Stop` button. The beeping should now stop.
```

Comment:
Technically this isn't true - our chargers will continue to beep even after pressing stop. I think the docs should match how charging actually works as closely as possible to reduce unnecessary worry.

Reply:
I've not seen this? Once they start beeping, pressing the button stops them? From what I remember anyway

Reply:
I thought they carried on beeping until they'd completed their full "beep duration" - could be wrong though. I'll check it tonight if I get time.

Reply:
After testing, it does stop it